extensions/che-theia-hosted-plugin-manager-extension/package.json

```diff
@@ -15,20 +15,36 @@
     "@eclipse-che/theia-remote-api": "^0.0.1"
   },
   "scripts": {
-    "prepare": "yarn clean && yarn build",
+    "prepare": "yarn clean && yarn build && yarn test",
     "clean": "rimraf lib",
     "format": "if-env SKIP_FORMAT=true && echo 'skip format check' || prettier --check '{src,tests}/**/*.ts' package.json",
     "format:fix": "prettier --write '{src,tests}/**/*.ts' package.json",
     "lint": "if-env SKIP_LINT=true && echo 'skip lint check' || eslint --cache=true --no-error-on-unmatched-pattern=true '{src,tests}/**/*.ts'",
     "lint:fix": "eslint --fix --cache=true --no-error-on-unmatched-pattern=true \"{src,tests}/**/*.{ts,tsx}\"",
     "compile": "tsc",
     "build": "concurrently -n \"format,lint,compile\" -c \"red,green,blue\" \"yarn format\" \"yarn lint\" \"yarn compile\"",
-    "watch": "tsc -w"
+    "watch": "tsc -w",
+    "test": "jest"
   },
   "license": "EPL-2.0",
   "theiaExtensions": [
     {
       "backend": "lib/node/hosted-plugin-manager-contribution-extension-backend-module"
     }
-  ]
+  ],
+  "jest": {
+    "clearMocks": true,
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/**/*.{ts,tsx}",
+      "!**/node_modules/**",
```

Comment:
hello, could you explain why we need to exclude tests or node modules folders ? AFAIK if we include only src folder we never have these

Reply:
It were additional precautions, just to be sure, that we won't get into coverage report any other directories. Removed unnecessary directories from the configs.